xml/System.Net.Quic/QuicConnection.xml

```diff
@@ -65,11 +76,22 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="errorCode">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="errorCode">An application-provided code with the reason for closure.</param>
+        <param name="cancellationToken">A cancellation token that can be used to cancel the asynchronous operation.</param>
+        <summary>Closes the connection with the application provided-code.</summary>
+        <returns>An asynchronous task that completes when the connection is closed.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Closing the connection is not graceful in regards to its streams. Calling <xref:System.Net.Quic.QuicConnection.CloseAsync(System.Int64,System.Threading.CancellationToken)> will immediately abort all streams associated with this connection. Make sure that all streams have been closed and all their data consumed before calling this method. Otherwise, all the data that was received but not yet consumed will be lost.
```

Comment:
```suggestion
Closing the connection is not graceful in regards to its streams. Calling <xref:System.Net.Quic.QuicConnection.CloseAsync(System.Int64,System.Threading.CancellationToken)> will immediately close all streams associated with this connection. Make sure that all streams have been closed and all their data consumed before calling this method. Otherwise, all the data that was received but not yet consumed will be lost.
```